src/interfaces/swig/SGBase.i

```diff
@@ -447,6 +472,52 @@ namespace shogun
 }
 
 %pythoncode %{
+def _internal_get_param(self, name):
+
+	try:
+		return self._get(name)
```

Comment:
Can we have something like that?
```
try:
  for f in (
    self._get,
    self._get_real
  ):
    f(name)
except:
  ..
```

Reply:
Yes, done!

Reply:
Ah wait, that's a good point putting the loop in the try except block